aws/requests.yaml

```diff
@@ -1,4 +1,9 @@
 releases:
+    - name: ">= 14.1.1"
```

Comment:
I think putting it in the `>= 15.0.0` bracket makes more sense -in general the `validate` CI job should succeed.

Reply:
changed :)